libs/list-types/sjp-public-list/src/pages/sjp-selection/en.ts

```diff
@@ -0,0 +1,6 @@
+export const en = {
+  title: "What do you want to view from Single Justice Procedure?",
+  back: "Back",
+  publicListPrefix: "SJP Public List – ",
```

Comment:
Is this used for the summary of pubs? If so I think the SJP lists should be using the standard summary of pubs rather than any specific logic for SJP